deployment.Jenkinsfile

```diff
@@ -53,13 +54,17 @@ node('master') {
     currentBuild.displayName = "Deploy $projectName " + version
 
     stage('Upload to S3 and/or Hockey') {
-        if(projectName.contains('Windows')) {
+        if (projectName.contains("Windows")) {
```

Comment:
inconsistent quotes.. let's use either double or single.. but not both :) (line 8 has single ones)

Reply:
Usually we only use double quotes in Jenkinsfiles when variables are inserted, so I should revert the double quotes.